supporter-product-data/src/test/scala/com/gu/lambdas/UpdateDynamoSpec.scala

```diff
@@ -0,0 +1,38 @@
+package com.gu.lambdas
+
+import com.gu.Fixtures
+import com.gu.model.Stage.DEV
+import com.gu.model.dynamo.SupporterRatePlanItem
+import com.gu.model.states.UpdateDynamoState
+import com.gu.test.tags.annotations.IntegrationTest
+import kantan.csv.ops.toCsvInputOps
+import kantan.csv.rfc
+import org.scalatest.flatspec.{AnyFlatSpec, AsyncFlatSpec}
+import org.scalatest.matchers.should.Matchers
+
+@IntegrationTest
+class UpdateDynamoSpec extends AsyncFlatSpec with Matchers {
+  "UpdateDynamoLambda" should "succeed" in {
+    UpdateDynamoLambda.writeToDynamo(
+      stage = DEV,
+      UpdateDynamoState(
+        filename = "New Subscriptions-2c92c0867728a2e801772ade71bb56a5",
+        recordCount = 50,
+        processedCount = 0
+      ),
+      new TimeOutCheck {
+        override def timeRemainingMillis = 9999999
+      }
+    ).map(_ =>
+      succeed
```

Comment:
how do we know the item got to dynamo?

Reply:
I've removed this test it was a duplicate